cabot/metricsapp/tests/fixtures/metrics_series.yaml

```diff
@@ -1,6 +1,4 @@
 error: False
-error_message:
```

Comment:
If there's no error, the series data wouldn't include error_message instead of having "error_message": None.

Doesn't belong in this PR though sorry!